### ISSUE TITLE
The importance of momentum encoder.

### ISSUE THREAD

**Issue by WangFeng18** [facebookresearch/dino#115]
Hi, thanks for providing the great work.
From Table 14, it seems that the momentum encoder is a significant component for vit-based model. 
Specifically, the SwAV achieve 71.8 with deit-s/16 model, which is lower than the performance on resnet-50, but when replacing the teacher network with momentum updated network, the performance seems higher than resnet-50, reaching 76.0.
So I want to know why momentum encoder is so important to vit-based models. Is there any insight to explain that?

**Comment by woctezuma** [comment]
A few excerpts from the article:

![Momentum encoder](https://i.imgur.com/Kd8r2wx.png)

![Momentum](https://i.imgur.com/rwMXrD0.png)

**Comment by mathildecaron31** [comment]
Hi @WangFeng18 
We have indeed observed that using a momentum network works particularly well when training ViT-based models compared to convnets. It is an empirical observation and we don't have much explanation why this is the case. We have made this observation for SwAV vs DINO and also for SimCLR vs MoCo (SimCLR and MoCo have comparable performance in our experiments with RN50 but MoCo works much better with ViT).
Hope that helps a bit.
